### ISSUE TITLE
Cesium release reminders

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium-concierge#80]
Right now we have calendar reminders to prep a release in the middle of the month and then do the release at the start of the month.

It would be great to replace this with a JSON file that is list of devs who do the release, and then the concierge can send all the reminders.

**Comment by pjcozzi** [comment]
@OmarShehata @ggetz is this on on the short-term radar?

**Comment by OmarShehata** [comment]
Yup! We've got a nice system that pulls the release schedule out of a YAML file so everyone on the team can see as well as open a PR to modify that config file. I think that'll make the process of "who's doing the release next" a lot simpler. 

I just need to write tests and open the PR.
